model/src/guild/permissions.rs

```diff
@@ -51,6 +51,9 @@ bitflags! {
         /// Allows for launching activities (applications with the `EMBEDDED`
         /// flag) in a voice channel.
         const START_EMBEDDED_ACTIVITIES = 1 << 39;
+        /// Allows for timing out users to prevent them from sending or reacting to
+        /// messages in chat and threads, and from speaking in voice and stage channels.
```

Comment:
we can add a link to the FAQ here as well: 
```suggestion
        /// Allows for timing out users to prevent them from sending or reacting
        /// to messages in chat and threads, and from speaking in voice and
        /// stage channels.
        ///
        /// See Discord's article on [Guild Timeouts].
        ///
        /// [Guild Timeouts]: https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ
```